### ISSUE TITLE
expanding metadata about datasets

### ISSUE THREAD

**Issue by pheyvaer** [LinkedDataFragments/Server.js#44]
Is there a way to expand the metadata that is shown on the index page about the datasets?

**Comment by rubensworks** [comment]
Extending the metadata with custom information you mean?

All this information is dynamically generated at the moment.
But I guess it should be possible to add support for this, by for example loading a global datasets metadata file, which will be included in the index datasource.

**Comment by pheyvaer** [comment]
Yes indeed custom information. For example, a URL to the complete dump.

**Comment by RubenVerborgh** [comment]
This is perfectly possible through a view extension, which can attach new triples to an existing view.

For instance, the [summary view](https://github.com/LinkedDataFragments/Server.js/blob/v2.2.2/lib/views/summary/TriplePattternFragmentsRdfView-Summary.js) adds an extra metadata triple. To activate, these views need to be listed in the configuration.
